server/passport-providers.js

```diff
@@ -159,5 +159,20 @@ module.exports = {
     scope: ['email'],
     link: true,
     failureFlash: true
+  },
+  'github-repo': {
```

Comment:
No, I tested and looked through the database, everything stays the same apart from the access token when you go to /link/github/repos/, it doesn't generate a new user identity entry... I looked around, and this is the best way we could find that escalates privileges with minimal impact to currently existing models and backend code... If I am missing something, please say, but this is the best way I could think of to escalate privlage